.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
-Please check the [FAQ](FAQ.md).  Issues without logs and details
-cannot be exploited, and will be closed.
+Issues without logs and details cannot be exploited, and will be closed.
+
+  - [ ] I have tried with the latest version of my channel (Stable or Edge)
```

Comment:
Not sure if users will easily conform to this (Stable versions in particular might not want to upgrade just to test, even though we would like them)

Reply:
Still, I think we should promote updating.  Do you feel I should remove that item?